service/controller/resource/ciliumnetpol/resource.go

```diff
@@ -113,3 +115,7 @@ func toCiliumNetworkPolicy(v interface{}) (*unstructured.Unstructured, error) {
 
 	return ciliumNetworkPolicy, nil
 }
+
+func hasClusterRoleBindingChanged(current *unstructured.Unstructured, desired *unstructured.Unstructured) bool {
```

Comment:
Maybe you want to rename to hasChanged ?

Reply:
Sure :sweat_smile: